website/content/v1.10/introduction/getting-started.md

```diff
@@ -1,323 +1,229 @@
 ---
-title: Getting Started
+title: Getting Started with a Local Talos Cluster 1.10
 weight: 30
-description: "A guide to setting up a Talos Linux cluster."
+description: "A guide to setting up a Talos cluster"
 ---
 
-This document will walk you through installing a simple Talos Cluster with a single control plane node and one or more worker nodes, explaining some of the concepts.
+This guide walks you through creating a simple Talos cluster with one control plane node and one or more worker nodes. 
 
-> If this is your first use of Talos Linux, we recommend the [Quickstart]({{< relref "quickstart" >}}) first, to quickly create a local virtual cluster in containers on your workstation.
->
-> For a production cluster, extra steps are needed - see [Production Notes]({{< relref "prodnotes" >}}).
+If you're looking to set up a cluster with multiple control plane nodes, see [Production Notes]({{< ref "prodnotes" >}}).
 
-Regardless of where you run Talos, the steps to create a Kubernetes cluster are:
+**New to Talos?** Start with [Quickstart]({{< ref "quickstart" >}}) to create a local virtual cluster on your workstation.
 
-- boot machines off the Talos Linux image
-- define the endpoint for the Kubernetes API and generate your machine configurations
-- configure Talos Linux by applying machine configurations to the machines
-- configure `talosctl`
-- bootstrap Kubernetes
+**Planning for production?** See [Production Notes]({{< ref "prodnotes" >}}) for additional requirements and best practices.
```

Comment:
let's use `relref` everywhere for consistency, and I think Hugo verifies all such links